packages/patrol/test/patrol_tester_test.dart

```diff
@@ -868,10 +873,17 @@ void main() {
           expect(find.text('top text').hitTestable(), findsNothing);
           expect(find.text('bottom text').hitTestable(), findsNothing);
 
-          await tester.scrollUntilExists(finder: find.text('top text'));
+          await tester.scrollUntilExists(
+            finder: find.text('top text'),
+            maxScrolls: 100,
```

Comment:
Did we have to change these tests because we modified the default?

If we really want to change the tests, then I'd prefer to change the tests so they pass with the default value.

Reply:
In this line it is unnecessary, but in other places - yes